pkg/sql/tests/rsg_test.go

```diff
@@ -271,8 +271,11 @@ func TestRandomSyntaxFunctions(t *testing.T) {
 					continue
 				}
 				switch lower {
-				case
-					"pg_sleep":
+				case "pg_sleep":
+					continue
+				case "st_frechetdistance":
```

Comment:
(how is sqlsmith not affected?)

Reply:
fair question... perhaps just because i haven't looked at it :O

i'll still merge this for now to get more signal out of RSG tests in the meantime, but will look into sqlsmith next.